### DISCUSSION TITLE
Poll: Which features are you missing the most in FalkorDB

### DISCUSSION THREAD

**Discussion by gkorland** [FalkorDB/FalkorDB#17] [Polls]
Collecting the feedback from [Welcome to FalkorDB Discussions!](https://github.com/orgs/FalkorDB/discussions/9)

**Comment by blockchain-art** [comment]
Am I right in thinking that RedisGears already enables `Subscribe to query changes (notifications)`?

**Reply by gkorland** [reply]
RedisGears allows you to register on a single notification but Falkor/RedisGraph is holding the all graph in a single Redis key so you can't get this level of resolution.

**Reply by blockchain-art** [reply]
Thanks. As an aside, is it possible to extend Nested JSON in properties to also include support for the map data type in properties

**Comment by JoonasC** [comment]
For me the feature I would like to see the most in Falkor would be the one I described in issue #13. As I see it, Falkor can keep the general design choice of being in memory (eg. no need to keep part of a graph on disk and part of it in memory), but it could offer more options for keeping some graphs on disk and only loading them into memory when needed.

**Comment by flowt-au** [comment]
Re: Nested JSON. I use RedisGraph as the primary database. Nodes represent "entities" (User, Task, Thread, Post, etc) The client app sends JSON packets which I need to manipulate into Cypher format. I can use toJson() to get the data back as JSON, but I still need to manually manage the instances where the JSON has nested objects within an entity, and where breaking the entity into sub-entities makes no logical sense. So, by "Nested JSON" what I mean is to be able to save the JSON data on the node properties as-is (ie quoted keys, objects, etc) and not have to manipulate it into something Cypher will cope with. And, I would still need to be able to use the FT search as well as the great pattern matching "querying" that Cypher offers. So, not asking for much, really. ;-)  
Thanks for considering the growing "wish list". Much appreciated.
